spec/features/login_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe 'login' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.